about.html

```diff
@@ -0,0 +1,166 @@
+      <div class="logo-img">
+        <img src="./images/logo-big.png" alt="Code with Raihan"/>
+      </div>
+      </div>
+    </section>
+
+    <!-- About Past-->
+    <section class="about-past">
+      <h1 class="program-title">See Our Past Bootcamp Activities</h1>
+      <p class="about-past-details">Take a look our past Bootcamp students who successfully made their career.</p>
+      <ul class="past-item-group">
+        <li class="past-item">
+          <div class="student-img">
+            <img src="./images/raihan.jpg" alt="Abu Raihan">
+            <h5 class="student-bootcamp">Fullstack Bootcamp 2022</h5>
+          </div>
+          <div class="student-info">
+            <h3 class="sudent-name">Abu Raihan</h3>
+            <p class="student-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque tenetur, commodi reprehenderit nihil neque ipsam, accusantium, ad dolorem architecto nostrum eveniet iste quis molestiae pariatur blanditiis ex id repudiandae cum!</p>
+          </div>
+        </li>
+
+        <li class="past-item">
+          <div class="student-img">
+            <img src="./images/raihan.jpg" alt="Abu Raihan">
+            <h5 class="student-bootcamp">Fullstack Bootcamp 2021</h5>
+          </div>
+          <div class="student-info">
+            <h3 class="sudent-name">Abu Raihan</h3>
+            <p class="student-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque tenetur, commodi reprehenderit nihil neque ipsam, accusantium, ad dolorem architecto nostrum eveniet iste quis molestiae pariatur blanditiis ex id repudiandae cum!</p>
+          </div>
+          </li>
+
+          <li class="past-item">
+            <div class="student-img">
+              <img src="./images/raihan.jpg" alt="Abu Raihan">
+              <h5 class="student-bootcamp">Fullstack Bootcamp 2021</h5>
+            </div>
+            <div class="student-info">
+              <h3 class="sudent-name">Abu Raihan</h3>
+              <p class="student-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque tenetur, commodi reprehenderit nihil neque ipsam, accusantium, ad dolorem architecto nostrum eveniet iste quis molestiae pariatur blanditiis ex id repudiandae cum!</p>
+            </div>
+          </li>
+          
+          <li class="past-item">
+            <div class="student-img">
+              <img src="./images/raihan.jpg" alt="Abu Raihan">
+              <h5 class="student-bootcamp">Fullstack Bootcamp 2020</h5>
+            </div>
+            <div class="student-info">
+              <h3 class="sudent-name">Abu Raihan</h3>
+              <p class="student-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eaque tenetur, commodi reprehenderit nihil neque ipsam, accusantium, ad dolorem architecto nostrum eveniet iste quis molestiae pariatur blanditiis ex id repudiandae cum!</p>
+            </div>
+          </li>
+      </ul>
+    </section>
```

Comment:
- [ ] While you did a great job with the past event section and your design looks stunning for the cards, unfortunately following the design guidelines is part of the requirements, it's like a real-life project where you get an guideline where you have to implement it as it is, kindly updates your cards to match the design guidelines 🙏 

**_Your design :_** 
![image](https://user-images.githubusercontent.com/89282221/203384213-af4ec3da-562a-4a1c-9bff-4e6fe6cb1e9c.png)

**_Required Design : _**
![image](https://user-images.githubusercontent.com/89282221/203384326-12479dfb-cd82-4ab5-92f4-dc7157650606.png)